autotest/gcore/tiff_read.py

```diff
@@ -2742,12 +2737,9 @@ def test_tiff_read_huge_implied_number_strips():
 ###############################################################################
 
 
+@pytest.mark.skipif(sys.platform == 'win32', reason='Runs super slow on some Windows configs')
 def test_tiff_read_many_blocks():
 
-    # Runs super slow on some Windows configs
```

Comment:
worth re-validating this? Was added 3 years ago in 2cdfe800bc6876ec3c60bb305802dc5b566bc6f8

Reply:
We'll see how CI likes it.

Reply:
Seems to be okay remove the skip.